config.py

```diff
@@ -0,0 +1,20 @@
+from aqt.addons import AddonManager
+
+
+class Config:
```

Comment:
I was playing around with lazy config loading and thought a config class might be good. But I'm not totally sure if your intention was lazy loading or something else. Would you mind adding a docstring to explain what it does?

Reply:
It's only extra code right now. I thought of lazy loading but my main reason for adding it now is just to centralize config loading/saving/updating in one place, and I thought maybe it can serve as the place to check for and report config errors, especially if more configurations are added in the future. I don't usually add such a class to my add-ons so I'm trying something new.

Reply:
I added a trivial docstring and made the config reflect user changes from the config dialog. Again, I'm not sure whether will this class be much useful in the future, so if you think nothing is gained by it I can remove it.

Reply:
No I liked it :)

Does the `_write()` function write back to the config.json file, or just the user's instance of the config?
I think the latter, right?

Reply:
It writes back to meta.json, which Anki uses for storing user's modifications to the config (config.json is never modified).